src/select/__tests__/select.test.tsx

```diff
@@ -450,6 +451,162 @@ describe.each([false, true])('expandToViewport=%s', expandToViewport => {
       wrapper.openDropdown();
       expect(wrapper.findDropdown({ expandToViewport })?.findOpenDropdown()).toBeFalsy();
     });
+
+    describe('Disabled item with reason', () => {
+      beforeEach(() => {
+        jest.spyOn(ReactDOM, 'createPortal').mockImplementation((element: any) => element);
```

Comment:
Same as in another thread https://github.com/cloudscape-design/components/pull/2409#discussion_r1656631973

Need a better solution